tests/src/test/scala/org/scalanative/bindgen/BindgenSpec.scala

```diff
@@ -38,10 +42,22 @@ class BindgenSpec extends FunSpec {
         val expected = new File(inputDirectory, testName + ".scala")
         val output   = new File(outputDir, testName + ".scala")
 
-        bindgen(input, testName, output)
+        val errs = bindgen(input, testName, output)
 
         assert(output.exists())
         assert(contentOf(output) == contentOf(expected))
+
+        input.getName match {
+          case "LiteralDefine.h" =>
+            assert(
+              errs == "Warning: integer value does not fit into 8 bytes: 18446744073709551615\n" +
+                "Warning: integer value does not fit into 8 bytes: 9223372036854775809")
+          case "OpaqueTypes.h" =>
+            assert(
+              errs == "Error: type definition for struct_undefinedStruct was not found.\n" +
+                "Error: Function useUndefinedStruct is skipped because it references an opaque type.")
+          case _ => assert(errs == "")
+        }
```

Comment:
To keep the tests clean, I would prefer to have checks for errors in a separate Spec.